src/hooks/useActiveProject.tsx

```diff
@@ -131,7 +139,11 @@ export const ActiveProjectContextProvider = ({
         ...hookReturnValue,
         setActiveProjectId,
         isLoading: !!(projectsResult.isLoading || useMeResult.isLoading),
-        isFetched: !!(projectsResult.isFetched && useMeResult.isFetched),
+        isFetched: !!(
+          projectsResult.isFetched &&
+          useMeResult.isFetched &&
+          isFetched
```

Comment:
The hook return value will lag behind the projectsResult.isFetched and useMeResult.isFetched by at least one render cycle. Adding our own flag to indicate when the hooks logic has successfully computed the result helps consumers of this hook understand when it is actually ready.